kubernetes/test/test_admissionregistration_api.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+
+"""
+    Kubernetes
+
+    No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)  # noqa: E501
+
+    The version of the OpenAPI document: release-1.17
+    Generated by: https://openapi-generator.tech
+"""
+
+
+from __future__ import absolute_import
+
+import unittest
+
+import kubernetes.client
+from kubernetes.client.api.admissionregistration_api import AdmissionregistrationApi  # noqa: E501
+from kubernetes.client.rest import ApiException
+
+
+class TestAdmissionregistrationApi(unittest.TestCase):
```

Comment:
Could you remove all the generated tests under `kubernetes/test` and just keep the two manually written tests https://github.com/kubernetes-client/python/tree/release-17.0/kubernetes/test? It's documented in the hot issue `#4`: https://github.com/kubernetes-client/python/blob/master/devel/release.md#hot-issues